openlibrary/plugins/openlibrary/code.py

```diff
@@ -423,53 +417,39 @@ class ia_js_cdn(delegate.page):
 
     def GET(self, filename):
         web.header('Content-Type', 'text/javascript')
-        raise web.HTTPError('200 OK', {}, fetch_ia_js(filename))
+        return web.ok(fetch_ia_js(filename))
 
 
 class serviceworker(delegate.page):
     path = '/sw.js'
 
     def GET(self):
         web.header('Content-Type', 'text/javascript')
-        try:
-            data = open('static/build/sw.js').read()
-            raise web.HTTPError('200 OK', {}, data)
-        except OSError:
```

Comment:
https://sentry.archive.org/organizations/ia-ux/issues/54065

Reply:
That happens only on ol-dev while the JS is building. Expected error for now! I do want this to appear in Sentry if it's missing, though.